bitcoin/src/blockdata/transaction.rs

```diff
@@ -1168,6 +1170,69 @@ impl From<&Transaction> for Wtxid {
     fn from(tx: &Transaction) -> Wtxid { tx.wtxid() }
 }
 
+/// Predicts the weight of a to-be-constructed transaction.
+///
+/// This function computes the weight of a transaction which is not fully known. All that is needed
+/// is an arbitrary set of TxIn and TxOut.
+///
+/// # Arguments
+///
+/// * `inputs` - a vector of inputs (TxIn).
+/// * `outputs` - a vector of outputs (TxOut).
+///
+/// # Usage
+///
+/// Determine the weight of a set of OutPoints (Utxos) before they are included in a transaction.
```

Comment:
Outpoints and UTXOs are completely different things.

Reply:
Oops this was a typo